### ISSUE TITLE
Add new prop for width to fast-layouts-react Column component to support SSR

### ISSUE THREAD

**Issue by k1mby** [microsoft/fast#2398]
Like to be able to provide the Column component inside fast-layouts-react with a specific width value to be used in place of window.innerWidth on the server side, would be great if we can have a prop added so we can pass that down.

The intended usage is inside getValueByBreakpoint(). If canUseDOM is false (we are on the server) we would want to pass identifyBreakpoint() the prop to use in place of window.innerWidth.

Thanks :)

**Comment by chrisdholt** [comment]
Would allowing a "default" breakpoint or something to that effect work? That would remove window.innerWidth from this entirely. The logic for that check would go to something like this with defaultProps set to 0:
```
    public static defaultProps: Partial<ColumnProps> = {
        defaultBreakpoint: 0
    }
   
    private getValueByBreakpoint<T>(breakpointSet: T[]): T {
        if (!canUseDOM()) {
            return breakpointSet[this.props.defaultBreakpoint];
        }

        const breakpoint: Breakpoint = identifyBreakpoint(window.innerWidth);

        return breakpointSet.slice(0, breakpoint + 1).pop();
    }
```

Then you could pass a corresponding breakpoint for where you would want to land rather than something relating to window.innerWidth.

**Comment by k1mby** [comment]
Sure that could work too - to give some context the scenario this would be used in is one where the server has been passed the client's window.innerWidth value and is now trying to render Column on the server side. 

Is it correct that we provide the values for breakpointSet via the span prop? Then I'm guessing if we have the client's window.innerWidth we can calculate what the breakpoint should be and pass that as a prop as you've suggested?

**Comment by Reza-Rajabi** [comment]
can I look into this one?

**Comment by chrisdholt** [comment]
@Reza-Rajabi Sure thing, PR's are welcome. I'd look to my comment above for some guidance on direction.

**Comment by Reza-Rajabi** [comment]
Thank you. Your comment is definitely helpful.

**Comment by chrisdholt** [comment]
closed by #2531 and #3517
